src/web/routes/application/address/postcode/adapters.js

```diff
@@ -2,14 +2,31 @@ const { compose, map, pick, prop, propOr } = require('ramda')
 const { OS_PLACES_ADDRESS_KEYS } = require('../constants')
 
 const RESULTS_PROP = 'results'
-
 const DELIVERY_POINT_ADDRESS_PROP = 'DPA'
+const OS_PLACES_TITLE_CASE_KEYS = OS_PLACES_ADDRESS_KEYS.filter(key => !['ADDRESS', 'POSTCODE'].includes(key))
+
+const toTitleCase = (str) => {
+  return str.replace(
+    /\b\w+/g,
+    function (txt) { return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase() }
+  )
+}
+
+const convertCase = (address) => {
+  const titleCase = compose(map(toTitleCase), pick(OS_PLACES_TITLE_CASE_KEYS))(address)
```

Comment:
As this uses `compose` I would move it outside of this function and then just call it here:

```
// Probably want a better name than this!!
const titleCaseAddress = compose(map(toTitleCase), pick(OS_PLACES_TITLE_CASE_KEYS))

const convertCase = (address) => {
  const titleCase = titleCaseAddress(address)
  ... 
}
```